### PR TITLE
Merge hotfix into develop 12/10/2022

### PR DESCRIPTION
**Pull request by haammar-ledger**
### 📝 Description

Merge the current hotfix branch to develop.

### ❓ Context

- **Impacted projects**: n/a 
- **Linked resource(s)**: n/a 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .changeset/pre.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "mode": "pre",
```
Comment (elbywan) [inline .changeset/pre.json:2]: This file should never exist outside of hotfix/release branches.

File: libs/ledgerjs/packages/types-live/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ledgerhq/types-live",
-  "version": "6.24.2",
+  "version": "6.24.3-hotfix.0",
```
Comment (elbywan) [inline libs/ledgerjs/packages/types-live/package.json:3]: wrong version, develop should never contain prereleases

File: .changeset/config.json [hunk 3]
```diff
@@ -12,5 +12,7 @@
   "access": "public",
   "baseBranch": "develop",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": [
+    "ledger-live-desktop"
```
Comment (haammar-ledger) [inline .changeset/config.json:16]: Not sure about this change :thinking: should I remove it?

Comment (elbywan) [inline .changeset/config.json:16]: Remove this.

Comment (elbywan) [reply .changeset/config.json:16]: Yep absolutely!

**Review by elbywan — APPROVED** [review]
👍

**Pull request merged**